proxmoxer/backends/command_base.py

```diff
@@ -111,15 +111,20 @@ def is_http_status_string(s):
             return re.match(r"\d\d\d [a-zA-Z]", str(s))
 
         if stderr:
-            # sometimes contains extra text like 'trying to acquire lock...OK'
-            status_code = next(
-                (
-                    int(line.split()[0])
-                    for line in stderr.splitlines()
-                    if is_http_status_string(line)
-                ),
-                500,
-            )
+            # assume if we got a task ID that the request was successful
+            task_id_pattern = re.compile(r'UPID:[\w-]+:[0-9a-fA-F]{8}:[0-9a-fA-F]{8}:[0-9a-fA-F]{8}:\w+:[\w._-]+:[\w@._-]+:\w*')
```

Comment:
the `.` in `\w._-` should be escape to match the literal dot/period instead of the regex interpretation of `.` which is a wildcard for any character.